.github/SECURITY.md

```diff
@@ -0,0 +1,7 @@
+# Security
+
+To report a security vulnerability please email security@dataverse.org as explained at https://guides.dataverse.org/en/latest/installation/config.html#reporting-security-issues
```

Comment:
We could also try using this new feature of GitHub: https://github.blog/changelog/2022-11-09-privately-report-vulnerabilities-to-repository-maintainers/

No rush, of course. 😄 No need to hold up with pull request.